### ISSUE TITLE
[UX] Menus: Move the "Configure menu" page at the top of the "Edit links" page.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2005]
I think that this one was mentioned before (by @jenlampton IIRC), but I cannot find any issue actually filed for it, so here's one...

With the menu title and description fields moved above the links table, this becomes a single form that takes only a single submit button and no additional page loads to edit aspects of the same thing (menu). We could then safely call this page either "Edit menu" or "Configure menu".

Making this a part of #1839 as well as #378. Here's how it looks in D8 (https://www.drupal.org/node/663946):

![d8-edit_menu_form-single_page](https://cloud.githubusercontent.com/assets/2423362/16578948/a72239ec-42e0-11e6-978b-cb69d7bca194.png)

**Comment by klonos** [comment]
In this follow-up https://www.drupal.org/node/2520232 they are discussing a few improvements/changes:

1. grouping the title and description fields inside a fieldset
2. making that fieldset collapsed by default and moving it below the list of menu links, before the submit button
3. changing the label of the title field to "**Menu** title" (instead of plain "Title")

I think that having the title and description fields between the list of links and the "Add link" button does indeed disconnect the two sections that are related, but if we moved the fieldset to the bottom of the page, we'd then require users to scroll more with menus that have many menu items. Also, that fieldset is not very likely to increase in height, while the menu item list would vary.

So, I'm perfectly fine with the change of the title field label and the grouping of the fields in a fieldset, but not a big fan of the idea of making that collapsed and moving it to the bottom of the form.

I would much better like it if the title and description stayed at the top of the page and the "Add link" button was moved bellow the fieldset that grouped them. This would I guess bring objections from those that think that the "Add x" buttons should be at the top of every page, but the alternative of moving those fields at the bottom of the form and requiring lots of scrolling is much worse UX if you ask me.

![backdrop-edit_menu_form-single_page](https://cloud.githubusercontent.com/assets/2423362/16580865/e51ca936-42e9-11e6-80bd-885d643ffd89.png)
